harness/tests/experiment/utils.py

```diff
@@ -145,6 +145,7 @@ def make_default_env_context(
         det_experiment_id="1",
         det_cluster_id="uuid-123",
         trial_seed=trial_seed,
+        test_mode=True,
```

Comment:
This seems like a very aggressive change... won't this prevent us from training or validating more than a single batch in all of our unit tests?